src/styles/mixins/forms.scss

```diff
@@ -35,63 +35,25 @@
   padding: 0.5em 0.75em;
   font-weight: normal;
   color: var.$grey-5;
-  background-color: #fafafa;
+  background-color: var.$grey-0;
 }
 
 @mixin form-input-focus {
   outline: none;
-  background-color: #fff;
+  background-color: var.$white;
 
   @include focus-outline;
   @include placeholder {
     color: var.$grey-5;
   }
 }
 
-@mixin form-input-error {
-  color: var.$error-color;
-  border-color: var.color-weight(var.$error-color, 300);
-  background-color: var.color-weight(var.$error-color, 50);
-
-  @include placeholder {
-    color: var.tint(var.$error-color, 5%);
-  }
-}
-
-@mixin btn {
-  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.15);
-
-  background: linear-gradient(var.$button-background-gradient);
-  display: inline-block;
-  font-weight: bold;
-  color: var.$button-text-color;
-  text-shadow: 0 1px 0 #fff;
-  border-radius: 2px;
-  border: 1px solid var.$grey-semi;
-  padding: 0.5em 0.9em;
-}
-
 @mixin btn-hover {
   box-shadow: 0 1px 0 rgba(0, 0, 0, 0.05);
   outline: none;
   color: var.$button-text-color;
   background: var.$button-background-start;
-  border-color: #bababa;
-}
-
-@mixin btn-active {
-  box-shadow: inset 0 1px 0 rgba(0, 0, 0, 0.1);
-  background: var.$button-background-end;
-  color: #424242;
-  border-color: #bababa;
-}
-
-@mixin btn-disabled {
-  box-shadow: none;
-  cursor: default;
-  background: #f0f0f0;
-  border-color: #cecece;
-  color: var.$grey-semi;
+  border-color: var.$grey-4;
```

Comment:
`btn-hover` should be able to go as soon as the new `TagEditor` lands as the existing tag editor is its only use in the app.